infrastructure/ecs.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_ecs_cluster" "app" {
+  name = "Vio"
+}
+
+resource "aws_ecs_task_definition" "app" {
+  family                   = "Vio-test-task"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  execution_role_arn       = "arn:aws:iam::460145940115:role/ecsTaskExecutionRole"
+
+  memory = "512"
+  cpu    = "256"
+
+  container_definitions = jsonencode([
+    {
+      name      = "App"
+      image     = "460145940115.dkr.ecr.eu-north-1.amazonaws.com/vio-app:latest"
+      cpu       = 0
+      memory    = 512
+      essential = true
+      command   = ["bundle", "exec", "puma"],
+      portMappings = [
+        {
+          containerPort = 4000
+          hostPort      = 4000
+        }
+      ]
+      environment = [
+        {
+          name  = "DATABASE_URL"
+          value = "postgres://postgres:postgres@viodb.c5htiu2ou7ka.eu-north-1.rds.amazonaws.com:5432/vio_production"
+        },
+        {
+          name  = "RAILS_ENV"
+          value = "production"
+        },
+        {
+          name  = "SECRET_KEY_BASE"
+          value = "8b14f710616bec48dda30facface86a7c00ebc2dee6e19d9b49e3ae8f8138790678dfc15152f8b3175b13b76124aad40e21a38b5c4c823a0db5918e2d08cfa37"
```

Comment:
`DATABASE_URL` and `SECRET_KEY_BASE` MUST be secrets that are pulled from AWS Secrets Manager rather than committed in git. I didn't have enough time to configure the secrets properly.